include/arch/x86/linker.ld

```diff
@@ -105,6 +106,15 @@ SECTIONS
 	} GROUP_LINK_IN(ROMABLE_REGION)
 
 	_image_text_end = .;
+
+#ifdef CONFIG_VERSION_HEADER
+	SECTION_PROLOGUE (version_header_section, (OPTIONAL),)
+	{
+		*(.version_header)
+		KEEP(*(".version_header*"))
```

Comment:
1. The second linkage wildcard subsumes the first, making the first one a noop.

2. Why is one KEEP() but the other not?